citesphere/src/test/java/edu/asu/diging/citesphere/core/service/impl/CitationManagerTest.java

```diff
@@ -395,4 +395,25 @@ public void test_getNotes_empty() throws GroupDoesNotExistException, CannotFindC
         List<ICitation> response = managerToTest.getNotes(user, GROUP_ID, EXISTING_ID);
         Assert.assertTrue(response.size() == 0);
     }
+    
+    
+    @Test
+    public void test_getCitationsByContributorUri() {
```

Comment:
this is just one test and it tests only the case when there are no results; what about the other ones?